### PR TITLE
{FRONT-END} BaseLayout Intitial merge

### PR DESCRIPTION
**Pull request by alexbo21**
BaseLayout Intital merge.

This pull request includes the basic styling included in the BaseLayout Package for Further Lunchtable Development. This includes
details like the Sidebar, Header, Profile Area and more in Future Commits and merges.

### CODE REVIEW

**Comment by alexbo21** [comment]
[{1} {FRONT-END} BaseLayout Intitial merge](https://trello.com/c/VvjVR3Is/23-1-front-end-baselayout-intitial-merge)

File: baseLayout/CSS/mainclient.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        position: absolute;
+        top:0px;
+        left:0px;
+    }
+
+    .headerProfilePicture{
+        width:70%;
+        /*  */
+        position: absolute;
+        right:15%;bottom:12.5%;
```
Comment (TheGreyDiamond) [inline baseLayout/CSS/mainclient.css:46]: Did you mean 
```
right: 15%;
bottom: 12.5%
```

File: baseLayout/CSS/mainclient.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+    .statusIndicator{
+        height:5.5pt;
+        width:5.5pt;
+        background:lightgreen;
+        border-radius:100%;
+        border:1pt solid white #e6d9cc;
+        /*  */
+        position: absolute;
+        bottom:13%;right:16%;
```
Comment (TheGreyDiamond) [inline baseLayout/CSS/mainclient.css:58]: Same as above

**Comment by alexbo21** [comment]
[[Front-end] 1](https://trello.com/c/VN4G1jaa/4-front-end-1)

File: baseLayout/CSS/mainclient.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        position: absolute;
+        top:0px;
+        left:0px;
+    }
+
+    .headerProfilePicture{
+        width:70%;
+        /*  */
+        position: absolute;
+        right:15%;  bottom:12.5%;
```
Comment (TheGreyDiamond) [inline baseLayout/CSS/mainclient.css:45]:
```
right:15%;
bottom:12.5%;
```

File: baseLayout/CSS/mainclient.css (hunk 1 above)
Comment (TheGreyDiamond) [reply baseLayout/CSS/mainclient.css]: Fixed it myself

**Comment by TheGreyDiamond** [comment]
Changed everything. It's fine now

**Comment by alexbo21** [comment]
Anything else?

**Comment by TheGreyDiamond** [comment]
Merge?

**Pull request merged**